### DISCUSSION TITLE
UNPKG URL?

### DISCUSSION THREAD

**Discussion by vinoth2442** [Milkdown/milkdown#732] [General]
Can we use milkdown as unpkg or CDN supported files ?
Shall we use this files without "module"?

**Comment by vinoth2442** [comment]
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Milkdown</title>
        
        <script type="module" src="https://cdn.jsdelivr.net/npm/@milkdown/exception@6.4.1/lib/index.es.min.js"></script>
        <script type="module" src="https://cdn.jsdelivr.net/npm/@milkdown/ctx@6.4.1/lib/index.es.min.js"></script>
        <script type="module">
            import { defaultValueCtx, Editor } from "https://cdn.jsdelivr.net/npm/@milkdown/core@6.4.1/lib/index.es.min.js";
            import { nord } from "https://cdn.jsdelivr.net/npm/@milkdown/theme-nord@6.4.1/lib/index.es.min.js";
            import { commonmark } from "https://cdn.jsdelivr.net/npm/@milkdown/preset-commonmark@6.4.2/lib/index.es.min.js";

            Editor.make().config((ctx) => {
                ctx.set(defaultValueCtx, "# Milkdown :heartpulse: Vanilla");
              })
              .use(nord)
              .use(commonmark)
              .create();
        </script>
    </head>
    <body>
    </body>
</html>

**Comment by Saul-Mirone** [comment]
It's not possible for now, see the discussion in https://github.com/Saul-Mirone/milkdown/issues/246.
